.github/workflows/test.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and test Jextract
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+
+  linux-x64:
+    runs-on: ubuntu-latest
```

Comment:
since you are downloading Ubuntu 18.04 binaries, wouldn't it be better to use 18.04 here? Also, 18.04 is a bit on the old side - wouldn't 20.04 be better?

https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-20.04.tar.xz

Reply:
Hmm, okay. I'll use that binary instead. There was no package for 20.04 for LLVM 13.0.1

Reply:
will not this break when `ubuntu-latest` binds to 22.04? Can we specify a specific ubuntu version?

Reply:
Ah, right. I'll set the version explicitly.